New package: MatricesForHomalg v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: MatricesForHomalg
- Repository: https://github.com/homalg-project/MatricesForHomalg.jl
- Created by: @mohamed-barakat
- Version: v0.1.0
- Commit: 8dff4d226dc3d6c535cd2cbeba65b4a438ae9cf3
- Reviewed by: @mohamed-barakat
- Reference: https://github.com/homalg-project/MatricesForHomalg.jl/commit/8dff4d226dc3d6c535cd2cbeba65b4a438ae9cf3#commitcomment-150965149
- Description: Matrices for the homalg project
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6f06aac5e64a0c4ef642cb73bf46434a7d73ebd903203285d19958bd1b928607b541ae8f356cb68907b740d07d08dffc8b8a326ecbdf5677f0aec1f6104fa513b649e41bd52896ca3c4c9999f0a0de0b11230a5530861594ef75e41e08f6daa3c2ef06735d2fc77c299c077e57d993e03bc23d14d183d5d0b1a5b335281a4b499771c43b0ab244080beac1523fd02c3b84040eedcd517dd70d5dc21f5b36bc84cff94d40c22c661f0c6b05b433406024be21b5a781db667661c5ac83220788603 -->

goerz commented:
[noblock] I would recommend adding a paragraph to the README to give some context for this package, including a link to the homalg project. Ideally, someone randomly coming across the repo should have a rough idea of what the purpose of the package is, just based on the information in the README

mohamed-barakat commented:
Yes, good idea, thank you very much.

How can I update this PR?

goerz commented:
[noblock] You can just comment `@JuliaRegistrator register` on the new commit, and it will update this PR. It's optional, though: an update to the README doesn't really affect the release, and anyone looking at the repo will see the latest version anyway. So you don't really have to do anything

JuliaRegistrator committed:
New package: MatricesForHomalg v0.1.0

UUID: 29b9b1b6-efa6-450e-8188-a5a2c25df071
Repo: https://github.com/homalg-project/MatricesForHomalg.jl.git
Tree: ca17941df274b95b935335199d50635caa221544

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
